client/jetpack-cloud/sections/partner-portal/issue-multiple-licenses-form/index.tsx

```diff
@@ -28,8 +28,8 @@ import {
 	getSelectedProductSlugs,
 } from 'calypso/state/partner-portal/products/selectors';
 import { APIProductFamilyProduct, PartnerPortalStore } from 'calypso/state/partner-portal/types';
+import useSubmitForm from '../issue-license-v2/hooks/use-submit-form';
```

Comment:
I suggest keeping this file in the same directory to minimize the number of files affected by this PR.

Reply:
We will eventually get rid of this directory once we ship this project during the cleanup. That's why I moved it here and copied the styles over.